### DISCUSSION TITLE
/admin/ hatası

### DISCUSSION THREAD

**Discussion by Ezagor-dev** [realsuayip/django-sozluk#37] [Q&A] [answered]
Merhabalar, 

Sözlüğü ubuntu, Nginx ve Gunicorn ile bağladım. Her sayfa çalışırken tek eksiğim /admin/ kısmı. 

siteadı.com/admin/ adresine gittiğimde tekrardan login sayfasına atıyor. Redirect loopa giriyor diyebilirim. Bunun için bir öneriniz var mı acaba ?  

<img width="620" alt="image" src="https://user-images.githubusercontent.com/45847677/152543900-af4c5f74-d1cc-4edd-bf95-2bb4b390d88d.png">

**Comment by realsuayip** [comment, marked as answer]
selamlar. admin sayfasına girebilmek için admin hesabı ile giriş yapmaniz gerekiyor.

**Reply by Ezagor-dev** [reply]
selamlar, ilginiz için çok teşekkürler. superuser yaratarak girişimi sağladım.
